tests/resources/wallet/test_thin_wallet.py

```diff
@@ -355,6 +356,7 @@ def test_token(self):
         add_blocks_unlock_reward(self.manager)
         token_name = 'MyTestToken'
         token_symbol = 'MTT'
+        token_info_version = TokenInfoVersion.DEPOSIT
```

Comment:
I'd rather rename all `token_info_version` to simply `token_version`, and the class `TokenInfoVersion` to `TokenVersion`.

Reply:
Thanks!